src/pages/playground/Playground.tsx

```diff
@@ -333,6 +337,36 @@ const Playground: React.FC<PlaygroundProps> = props => {
     state => state.workspaces.playground.externalLibrary
   );
 
+  const { uuid } = useParams();
+
+  const handleURL = () => {
+    if (uuid) {
```

Comment:
Nit: Don't use a closure for handlers if possible as it makes the data flow less straightforward. Please use dependency injection to make it more "pure".

Reply:
changed uuid as a parameter of handleURL function.